octangroup/git-cafe-exercise
# Hotfix/contact telephone

# Exercise 2 - Hotfix

## Description

This pull request addresses a hotfix in the `git-cafe-exercise` repository by updating the telephone number on the `index-4.html` page from `+1 800 603 6035` to `+1 800 659 6035`.

#### What does this PR do?

This PR applies a hotfix to the `git-cafe-exercise` repository by updating the telephone number on the `index-4.html` page to `+1 800 659 6035`.

#### Description of Task Completed

- Created a new hotfix branch named `hotfix/contact-telephone`.
- Modified the `index-4.html` file to change the telephone number to `+1 800 659 6035`.

#### How should this be manually tested?

To manually test this PR:
1. Clone the repository to your local machine.
2. Checkout the `hotfix/contact-telephone` branch.
3. Open the `index-4.html` file in a web browser.
4. Verify that the telephone number is displayed as `+1 800 659 6035` on the contact page.

#### Any background context you want to provide?

This hotfix is necessary to correct the incorrect telephone number displayed on the contact page of the `git-cafe-exercise` website.
